extensions/amp-list/0.1/amp-list.js

```diff
@@ -495,7 +495,8 @@ export class AmpList extends AMP.BaseElement {
       this.templates_.findAndRenderTemplateArray(this.element, array)
           .then(results => this.updateBindings_(results))
           .then(elements => this.render_(elements, current.append))
-          .then(() => this.maybeRenderLoadMoreTemplates_(payload))
+          .then(() => this.loadMoreEnabled_
```

Comment:
signal: the `.then(() => booleanValue && this.promiseReturningFunction())` works, but could be leaning in the direction of being unhelpfully terse

Reply:
I agree. Let me refactor this then instead of reusing this odd pattern.

Reply:
sg, feel free to refactor in a separate PR if this fixes a bug for your feature

Reply:
Done.